src/cf-pagination/usage.md

```diff
@@ -45,92 +40,98 @@ Default pagination consists of "Previous" and "Next" links, styled as buttons, a
 To enable the component to jump directly to the paginated content, place `#pagination_content` directly above your paginated content.
 
 ### Example
-<div id="pagination_content"></div>
+<div id="o-filterable-list-controls"></div>
 
 <!-- Paginated content here -->
 
-<nav class="pagination">
-    <a class="btn btn__super pagination_prev" href="#pagination_content">
-        <span class="btn_icon__left cf-icon cf-icon-left"></span>
-        Previous
+<nav class="m-pagination" role="navigation" aria-label="Pagination">
+    <a class="btn
+              m-pagination_btn-prev"
+       href="?page=1#o-filterable-list-controls">
```

Comment:
See the note above about why it exists, but I think this should remain `pagination_content` (or `m-pagination-content`). It may apply to situations where you're paginating things that aren't part of a filterable list.